core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -860,26 +870,37 @@ public void run() {
                                 .setContentIntent(pIntent)
```

Comment:
Just wondering, wouldn't it make sense to ``.setOngoing(true)`` [(Link)](https://developer.android.com/reference/android/app/Notification.Builder.html#setOngoing(boolean)) only when playing, else set it to false?
That way, we wouldn't need the X close button and users could just swipe the notification away when paused